app/src/main/java/com/example/concordia_campus_guide/view_models/PathsViewModel.java

```diff
@@ -143,6 +141,21 @@ private void addIndoorDescriptionToList(WalkingPoint startWalkingPoint, WalkingP
                     addCardToList("Walk towards classroom " + endWalkingPoint.getPlaceCode(), "Classroom");
                 }
                 break;
+            case PointType.WASHROOM:
```

Comment:
Is it normal that these are hardcoded? Does it mean it won't be translated?

Reply:
That's true! I'll add them to the resources but I need someone to translate them for me :P

Reply:
Perfect! I'll help you out!